.buildkite/pipeline.yml

```diff
@@ -10,6 +10,12 @@ steps:
       - label: ":construction_worker: Build stack installers / Snapshot"
         command: ".buildkite/scripts/build.ps1"
         key: "build-snapshot"
+        if: |
+          // Only run when triggered from Unified Release or via PRs targetting non-main, since there is no valid MANIFEST_URL for snapshot/main branch.
+          (
+            ( build.env("BUILDKITE_PULL_REQUEST") != "false" && build.env("BUILDKITE_PULL_REQUEST_BASE_BRANCH") =~ /^[0-9]+\.[0-9]+\$/ ) ||
```

Comment:
This won't run on prs targeting main.

Reply:
well spotted. Additionally, `main` is missing below too. I am closing this PR in favor of https://github.com/elastic/elastic-stack-installers/pull/248